docs/faqs.adoc

```diff
@@ -0,0 +1,43 @@
+= Kadalu Storage FAQs
+
+== Is Kadalu Storage the same as GlusterFS except for the management layer?
+
+Kadalu Storage is an opinionated Storage solution. It uses only the core filesystem from GlusterFS. Kadalu Storage brings good parts of GlusterFS with added enhancements like improved Quota, Management layer and others.
+
+Along with the additional features, the default configurations/values will differ. All the enhancements available with Kadalu Storage are also Open Source.
```

Comment:
Can we add a line saying there will be more intuitive CLI and syntax.

Reply:
Done